src/server/socketHandlers/scConnectionHandler.js

```diff
@@ -53,5 +54,7 @@ export default function scConnectionHandler(exchange) {
       authToken.tms = tms;
       socket.setAuthToken(authToken);
     }
+    // Publish current app version to notify client of possible change
+    socket.exchange.publish('version', APP_VERSION);
```

Comment:
this would publish the version to all subscribed clients every time a new connection is established. i'm not sure that's what you're after